apps/artist/queries/show_fragment.coffee

```diff
@@ -9,9 +9,7 @@ module.exports =
         name
       }
     }
-    location {
-      city
-    }
+    city
```

Comment:
Support the new way to get the `city`, either from Galaxy or Artsy. The [display helper](https://github.com/artsy/force/blob/93fc7773a6d509955bc1e476b5fa28a8f8ee80cf/components/show_cell/helpers.coffee#L14-L17) for the show cell is already updated to support all the styles of querying for locations.